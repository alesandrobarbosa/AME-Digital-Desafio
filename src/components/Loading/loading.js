import React from 'react';

import './loading.scss';
let loaderImg = require('../../view/img/loader.png');


const Loading = () => {
  return (
    <div className="container">
      <img className="container__loader" src={loaderImg} alt="Loading..." />
      Loading...
    </div>
  );
}

export default Loading;