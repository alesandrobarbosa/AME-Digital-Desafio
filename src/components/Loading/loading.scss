@import '../../view/styles/styles.scss';

.container {
    flex-flow: column;
    max-width: 400px;
    margin: auto;
    justify-content: center;
    font-family: $fontStyle;
    color: $loadingTxtColor;
    text-align: center;
    font-size: 1.5em;

    &__loader {
        margin: auto;
        width: 42%;
        padding: 20px;
    }

}