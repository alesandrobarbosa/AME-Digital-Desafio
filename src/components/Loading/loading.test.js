import React from 'react';
import renderer from 'react-test-renderer';

import Loading from './loading';

it('Loading renders should render', () => {
    const tree = renderer.create(<Loading></Loading>).toJSON()
    expect(tree).toMatchSnapshot()
});
