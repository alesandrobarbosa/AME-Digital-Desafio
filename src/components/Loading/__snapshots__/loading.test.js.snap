// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#LAODING COMPONENT ==> Loading should renders without crashing  1`] = `
<div
  className="container"
>
  <img
    alt="Loading..."
    className="container__loader"
    src="loader.png"
  />
  Loading...
</div>
`;
